redaxo/src/addons/structure/plugins/content/pages/modules.modules.php

```diff
@@ -91,10 +92,7 @@
         }
 
         $error = rex_i18n::msg('module_cannot_be_deleted', $modulname);
-
-        if ('' != $moduleInUseMessage) {
-            $error .= '<ul>' . $moduleInUseMessage . '</ul>';
-        }
+        $error .= '<ul>' . $moduleInUseMessage . '</ul>';
```

Comment:
die `for` schleife wird immer durchlaufen, da `$del->getRows() < 0` ist (siehe äußerer check)